docs/reST/ref/sprite.rst

```diff
@@ -634,12 +634,25 @@ Sprites are not thread safe. So lock them yourself if using threads.
       | :sl:`sets the threshold in milliseconds`
       | :sg:`set_timing_treshold(time_ms) -> None`
 
+      DEPRECATED: Use set_timing_threshold() instead.
+
+      .. deprecated:: 2.1.1
+
+      .. ## LayeredDirty.set_timing_treshold ##
+
+   .. method:: set_timing_threshold
+
+      | :sl:`sets the threshold in milliseconds`
+      | :sg:`set_timing_threshold(time_ms) -> None`
+
       Default is 1000./80 where 80 is the fps I want to switch to full screen
       mode.  This method's name is a typo and should be fixed.
```

Comment:
not anymore.

Reply:
This whole doc has problems.

> Default is 1000./80 where 80 is the fps I want to switch to full screen mode. This method's name is a typo and should be fixed.

What does that even mean?

> It uses the dirty flag technique and is therefore faster than the pygame.sprite.RenderUpdatesGroup sub-class that tracks dirty updates. if you have many static sprites. It also switches automatically between dirty rect update and full screen drawing, so you do no have to worry what would be faster.

So you do **not** have to worry what would be faster

Reply:
Okay I will fix these.